src/http/server_context.rs

```diff
@@ -1,19 +1,34 @@
+use std::sync::Arc;
+
+use async_graphql::dataloader::{DataLoader, NoCache};
 use async_graphql::dynamic;
 use derive_setters::Setters;
 
 use crate::blueprint::Blueprint;
 use crate::config::Server;
-use crate::http::HttpClient;
+use crate::http::{HttpClient, HttpDataLoader};
 
-#[derive(Clone, Setters)]
+#[derive(Setters, Clone)]
 pub struct ServerContext {
   pub schema: dynamic::Schema,
   pub http_client: HttpClient,
   pub server: Server,
+  pub data_loader: Arc<Option<DataLoader<HttpDataLoader, NoCache>>>,
```

Comment:
```suggestion
  pub data_loader: Arc<DataLoader<HttpDataLoader, NoCache>>,
```

We don't need to wrap in Option because we can initiate the DL always on server start. But only pass the request to DL if it's enabled.